_articles/system76-software.md

```diff
@@ -12,6 +12,10 @@ section: hardware-drivers
 
 ---
 
+# Notes about these instructions
```

Comment:
NOTE: These instructions were tested on a Galago Pro (galp3-b) and Pangolin (pang10). Neither system have NVIDIA GPUs, so this process doesn't go over installing the NVIDIA driver (system76-driver-nvidia).
Due to these limitations, switchable graphics may not work on NVIDIA systems.